summary_workflows/identification/specification/I_benchmark_specification.md

```diff
@@ -151,36 +200,45 @@ The OpenEBench assessment file contains the following attributes:
 | `Sensitivity_50nt` | `float` | N/A | Sensitivity of PAS identification compared with orthogonal dataset; Sensitivity = (TP/(TP+FN)); calculated for 50 nt distance threshold |
 | `Sensitivity_100nt` | `float` | N/A | Sensitivity of PAS identification compared with orthogonal dataset; Sensitivity = (TP/(TP+FN)); calculated for 100 nt distance threshold |
-| `FPR_10nt` | `float` | N/A | False Positive Rate of PAS identification compared with orthogonal dataset; FPR = FP/(FP+TN); calculated for 10 nt distance threshold |
-| `FPR_50nt` | `float` | N/A | False Positive Rate of PAS identification compared with orthogonal dataset; FPR = FP/(FP+TN); calculated for 50 nt distance threshold |
-| `FPR_100nt` | `float` | N/A | False Positive Rate of PAS identification compared with orthogonal dataset; FPR = FP/(FP+TN); calculated for 100 nt distance threshold |
-| `AUC` | `float` | N/A | Area under the curve (AUC) of receiver operating characteristic (ROC) calculated from TPR and FPR values for all distance thresholds between 0 nt and 100 nt with 10 nt intervals. |
+| `AUC` | `float` | N/A | Area under the curve (AUC) of Precision-Recall curve calculated from Precision and Recall (Sensitivity) values for all a range of distance thresholds |
+| `Proportion_in_3UTR` | `float` | % | Proportion of PAS identified from RNA-seq that were assigned to 3'-UTRs |
+| `Multi-matched_10nt` | `float` | % | Proportion of PAS identified from RNA-seq that were matched to multiple sites in grount truth; calculated for 10 nt distance threshold |
+| `Multi-matched_50nt` | `float` | % | Proportion of PAS identified from RNA-seq that were matched to multiple sites in grount truth; calculated for 50 nt distance threshold |
+| `Multi-matched_100nt` | `float` | % | Proportion of PAS identified from RNA-seq that were matched to multiple sites in grount truth; calculated for 100 nt distance threshold |
+| `Genes_correct_PAS` | `float` | % | Percentage of genes with correctly identified number of PAS |
 
 #### Output 2
 
-The OpenEBench consolidation file specifies visualization types (2D scatter plot, barplot), descriptions of metrics used for X and Y axis and (X,Y) value pairs for each challenge.
+The OpenEBench aggregation file contains information required to produce one plot.
+
+#### Output 3
+
+The OpenEBench consolidation file contains all the information about the new benchmarking run and specifies visualization types (2D scatter plot, barplot), descriptions of metrics used for X and Y axis and (X,Y) value pairs for each challenge and challenge participants.
 
 [//]: # (References)
   
 [in1]: ./example_files/input1.bed
 [in2]: ./example_files/input2.bed
+[in3]: ./example_files/input3.gtf
 [assessment_out]: ./example_files/assessment_out.json
+[aggregation_out]: ./example_files/aggregation_out.json
 [consolidation_out]: ./example_files/consolidation_out.json
```

Comment:
This file doesn't exist

Reply:
I removed the consolidation file example.